packages/client-kinesis-browser/model/ServiceMetadata.ts

```diff
@@ -12,4 +12,4 @@ export const ServiceMetadata: _ServiceMetadata_ = {
     targetPrefix: 'Kinesis_20131202',
     uid: 'kinesis-2013-12-02'
 };
-export const clientVersion: string = '0.1.0-preview.2';
\ No newline at end of file
+export const clientVersion: string = '0.1.0-preview.3';
```

Comment:
I'm wondering why GitHub see this as a diff. The metadata is already up-to-date in master.
https://github.com/aws/aws-sdk-js-v3/blob/master/packages/client-kinesis-browser/model/ServiceMetadata.ts#L15

Reply:
I find many of the diffs come the last commit: https://github.com/aws/aws-sdk-js-v3/commit/51eb26f09f0621dc4bb51b2627562c6badac815d. Maybe you can rebase before open this PR?

Reply:
This is fixed now.